packages/runtime-core/src/componentPublicInstance.ts

```diff
@@ -202,16 +209,23 @@ export type ComponentPublicInstance<
   MakeDefaultsOptional extends boolean = false,
   Options = ComponentOptionsBase<any, any, any, any, any, any, any, any, any>,
   I extends ComponentInjectOptions = {},
-  S extends SlotsType = {}
+  S extends SlotsType = {},
+  Attrs extends AttrsType = Record<string, unknown>,
+  PropsAttrs = HasDefineAttrs<Attrs> extends true
```

Comment:
Could you please explain what `PropsAttrs` is used for?

Reply:
Comments added.